smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -45,6 +45,11 @@ contract IPublicLock is IERC721Enumerable {
     uint keyPrice
   );
 
+  event TokenAddressChanged(
```

Comment:
This is redundant with PricingChanged

Reply:
Yes, this is an artifact of the first approach I took.